site/includes/footer.php

```diff
@@ -1,32 +1,32 @@
-        <footer>
+        <footer class="p-2 py-md-5 px-lg-5">
```

Comment:
I guess this also needs `container-fluid` to work with the grid (rows should be nesting in a container type class). Then some x padding for small until it hits large, maybe px-sm-4 or 5?